test.sh

```diff
@@ -781,3 +781,52 @@ no_more_events=$(curl -sSfg -u $SK: $HOST/v1/events?starting_after=$last_event \
 zero_events=$(curl -sSfg -u $SK: $HOST/v1/events?starting_after=$last_event \
                  | grep -oE '^  "data": \[\]')
 [ -n "$zero_events" ]
+
+# create a pending payout
+payout=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts \
+       -d amount=1100 \
+       -d currency=eur \
+  | grep -oE 'po_\w+' | head -n 1)
+
+# check correct status
+payout_status=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts/$payout \
+  | grep -oE '"status": "pending",')
+[ -n "$payout_status" ]
+
+# cancel the payout
+curl -sSfg -u $SK: $HOST/v1/payouts/$payout/cancel \
+  -X POST
```

Comment:
You don't want the `-S` and `-f` options here. Especiall `-f`, which is used to fail silently on server errors.

Better to use:
```suggestion
curl -sg -u $SK: $HOST/v1/payouts/$payout/cancel -X POST
```